Userland/Utilities/markdown-check.cpp

```diff
@@ -243,7 +243,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     Core::ArgsParser args_parser;
     Vector<StringView> file_paths;
+    StringView base_path;
     args_parser.add_positional_argument(file_paths, "Path to markdown files to read and parse", "paths", Core::ArgsParser::Required::Yes);
+    args_parser.add_option(base_path, "System base path (default: \"/\")", "base", 'b', "path");
```

Comment:
This should not be introduced in its own commit if nothing is using it yet

Also typo in the commit message: "This is important to [missing word?] which files"